src/modules/sitemap/sitemap.types.ts

```diff
@@ -0,0 +1,19 @@
+export interface SitemapItemInfo {
+	loc: string;
+	changefreq: string;
+	lastmod?: string;
+	priority?: number;
+}
+
+export interface SitemapConfig {
+	value: PathInfo[];
+	name: string;
+	updated_at?: string;
+	created_at?: string;
+}
+
+export interface PathInfo {
```

Comment:
REQ: rename **SitemapPathInfo**. Please use the provided naming conventions as used in other modules types. Always use name of the module + append according to the implementation.

Usecase: what if we define PathInfo in another module and we have to use both in the same module when imported? Then we would have a problem (yes we can alias it but its better to do it right the first time).